app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -14,7 +14,10 @@ div.panel.panel-default
           td = @submission.grade
         tr
           th = @submission.experience_points_record.class.human_attribute_name(:points_awarded)
-          td = @submission.points_awarded
+          - if can?(:grade, @submission)
+            = f.input :points_awarded, wrapper_tag: :td, label: false
```

Comment:
simple_form does generate a td tag. You are saying that we should keep the td and the containing div for the input field separate?